input/pages/usecases.xml

```diff
@@ -296,7 +296,7 @@ code=69548-6&_include=Observation:
     <a name="gettingThruGroupers"></a>
   <h4>Iterating through has-member and derived-from</h4>
   <p>
-  This example is using a Genomics Diagnostic Resource as a base to demonstrate iteration through has-member and dervied-from in the same query. Here a specific instance of diagnostic report is being queried for all the connected genomic observation profiles. If the DiagnosticReport.results are only grouping observations or panels, this query would ensure finding the leaf node observations</p>
+  This example is using a Genomics Diagnostic Resource as a base to demonstrate iteration through has-member and derived-from in the same query. Here a specific instance of diagnostic report is being queried for all the connected genomic observation profiles. If the DiagnosticReport.results are only grouping observations or panels, this query would ensure finding the leaf node observations</p>
```

Comment:
If we are fixing things, we should rename:

has-member -> hasMember
derived-from -> derivedFrom

Reply:
We are talking about queries. The query parameter is different

Reply:
ha ha, good point.  not an issue.